### PR TITLE
Fixed parse env variable EMAIL_PROVIDER_SMTP_REJECT_UNAUTHORIZED

### PR DESCRIPTION
**Pull request by SpeedyGonzaless**
Fixed parse from env, as env variable is always string and zod cannot parse string to boolean - it is needed to convert it to boolean

### CODE REVIEW

File: packages/services/emails/src/environment.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ const SMTPEmailModel = zod.object({
   EMAIL_PROVIDER_SMTP_PORT: NumberFromString,
   EMAIL_PROVIDER_SMTP_AUTH_USERNAME: zod.string(),
   EMAIL_PROVIDER_SMTP_AUTH_PASSWORD: zod.string(),
-  EMAIL_PROVIDER_SMTP_REJECT_UNAUTHORIZED: zod.boolean().optional(),
+  EMAIL_PROVIDER_SMTP_REJECT_UNAUTHORIZED: zod.string().optional(),
```
Comment (n1ru4l) [inline packages/services/emails/src/environment.ts:59]: Please use `emptyString(zod.union([zod.literal('0')], zod.literal('1')]).optional())`

**Review by n1ru4l — CHANGES_REQUESTED** [review]
Thank you for this contribution! I added a comment.

File: packages/services/emails/src/environment.ts (hunk 1 above)
Comment (SpeedyGonzaless) [reply packages/services/emails/src/environment.ts]: I got it, fixed to stay consistent

**Review by n1ru4l — APPROVED** [review]

**Comment by n1ru4l** [comment]
@SpeedyGonzaless thank you

**Comment by SpeedyGonzaless** [comment]
> @SpeedyGonzaless thank you

Can you merge pls? I want to have a final working build, to finish helm chart with self-hosted graphql-hive and publish it

File: packages/services/emails/src/environment.ts [hunk 2]
```diff
@@ -156,7 +158,7 @@ const emailProviderConfig =
           pass: email.EMAIL_PROVIDER_SMTP_AUTH_PASSWORD,
         },
         tls: {
-          rejectUnauthorized: email.EMAIL_PROVIDER_SMTP_REJECT_UNAUTHORIZED,
+          rejectUnauthorized: !(email.EMAIL_PROVIDER_SMTP_REJECT_UNAUTHORIZED === '0'),
```
Comment (kamilkisiela) [inline packages/services/emails/src/environment.ts:161]: it's `true` by default, right?

Comment (SpeedyGonzaless) [reply packages/services/emails/src/environment.ts:161]: Yes, by default it's true
https://nodejs.org/api/tls.html#tlscreateserveroptions-secureconnectionlistener
`rejectUnauthorized - If not false the server will reject any connection which is not authorized with the list of supplied CAs. This option only has an effect if requestCert is true. Default: true.`

**Pull request merged**